suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -665,3 +658,13 @@ export const selectIsDeviceAuthenticityFulfilled = (
         (deviceAuthenticity.error === 'CA_PUBKEY_NOT_FOUND' && deviceAuthenticity.configExpired) // CA_PUBKEY_NOT_FOUND with configExpired is temporary allowed and just logged to Sentry
     );
 };
+
+export const selectIsSelectedDeviceImported = (state: DeviceRootState) => {
+    const device = selectDevice(state);
+    return device?.id === HIDDEN_DEVICE_ID;
+};
+
+export const selectIsSelectedDeviceAuthorized = (state: DeviceRootState) => {
```

Comment:
We already have very similar selector `selectIsDeviceConnectedAndAuthorized` Can not you use that one? If they are actually different, then is the naming confusing.

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4